macros/CompatEdge.ejs

```diff
@@ -190,6 +195,36 @@ if (!release) {
         release = {
             version: str20OrEarlier
         };
+    } else if (Math.floor(Number(edgeHTMLVersion)) >= 10049) {
+        if (Math.floor(Number(edgeHTMLVersion)) >= 10240) {
+            // attempt to handle build numbers
+            var HTMLVersion = 0;
+            var edgeVersion = 0;
+            var BuildNumber = 0;
+            var releaseFound = false;
+            for (var key of releases) {
```

Comment:
This does not work for me because `releases` is not iterable. That worries me because it suggest that this code hasn't been tested at all.